superset/views/core.py

```diff
@@ -1175,11 +1186,9 @@ def explore(self, datasource_type=None, datasource_id=None):
             return redirect(datasource.default_endpoint)
 
         # slc perms
-        slice_add_perm = security_manager.can_access("can_add", "SliceModelView")
+        slice_add_perm = security_manager.can_access("can_write", "Slice")
```

Comment:
I think on SliceModelView , you have set "class_permission_name = "Chart", so not sure if this is supposed to be "Slice"